db/silkworm/etl/collector.cpp

```diff
@@ -96,9 +100,16 @@ void Collector::load(silkworm::lmdb::Table* table, LoadFunc load_func, unsigned
         if (load_func) {
             for (const auto& transformed_etl_entry : load_func(etl_entry)) {
                 table->put(transformed_etl_entry.key, transformed_etl_entry.value, flags);
+                log_tracker += transformed_etl_entry.size();
             }
         } else {
             table->put(etl_entry.key, etl_entry.value, flags);
+            log_tracker += etl_entry.size();
+        }
+        // Oncde in a while updates the user on current progress
+        if (!op_name.empty() && log_tracker >= kLogInterval) {
```

Comment:
This is nice but actually does not give the perception of the progress we're at.
Maybe tracking **overall** size of collector and report a percentage of the load is better ?